app/components/search-facet-daterange/component.js

```diff
@@ -45,11 +46,23 @@ export default Ember.Component.extend({
 
     },
 
-    filterUpdated: Ember.observer('filter', function() {
-        let filter = this.get('filter');
-        if (filter) {
-            let key = this.get('key');
-            let { start, end } = invertDateRangeFilter(key, filter);
+    statePrevious: [],
+    // this isn't getting triggered
```

Comment:
@aaxelb this isn't getting triggered when the state changes. Do you have an idea of why? Would `state` being an object impact the observer getting triggered?